plugin/evm/message/handler.go

```diff
@@ -29,6 +30,11 @@ func (NoopMempoolGossipHandler) HandleTxs(nodeID ids.NodeID, _ TxsGossip) error
 	return nil
 }
 
+func (NoopMempoolGossipHandler) HandleSignedOrders(nodeID ids.NodeID, _ SignedOrdersGossip) error {
+	log.Debug("dropping unexpected SignedOrdersGossip message", "peerID", nodeID)
+	return nil
+}
```

Comment:
what does this do?

Reply:
it's the no-op handler, it's used in unit tests